Automattic/jetpack
# Jetpack Sync: Add Updates module in required Sync modules

Adds Sync `Updates` module in required Sync modules to ensure `Updates` module will always be present, no matter the consumer configuration

#### Changes proposed in this Pull Request:

* Data_Settings: Added `Automattic\\Jetpack\\Sync\\Modules\\Updates` in MUST_SYNC_DATA_SETTINGS `jetpack_sync_modules` 

#### Other information:

#### Jetpack product discussion


p9dueE-66S-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
On a JN site with only standalone Protect active:
- Use WP Downgrade plugin to upgrade/downgrade WP core
- Visit the JP Debugger after each change and make sure the correct WP version is reported there
